pkg/sql/backfill.go

```diff
@@ -802,7 +802,11 @@ func TruncateInterleavedIndexes(
 		// All the data chunks have been removed. Now also removed the
 		// zone configs for the dropped indexes, if any.
 		if err := db.Txn(ctx, func(ctx context.Context, txn *kv.Txn) error {
-			return RemoveIndexZoneConfigs(ctx, txn, execCfg, table.GetParentID(), indexes)
```

Comment:
I'm certain that `table.GetParentID()` instead of `table.GetID()` is a bug here but found no way to test it. It only involves zone configs on interleaved indexes so I'm not super motivated to do more.